assets/js/disable-multiple-submit.js

```diff
@@ -15,9 +14,9 @@ $(document).ready(function disableMultipleSubmit() {
 
     $inputSubmit.data('throttle', true);
 
-    setTimeout(() => {
+    setTimeout(function enable() {
       $inputSubmit.data('throttle', false);
-    }, throttleTimeout);
+    }, submitThrottleTimeout);
```

Comment:
I prefer throttleTimeout as a variable

I feel like verbs suggest it's a function for me that is. I looked for a function, maybe it's just me

Reply:
Good note, thank you!